deployment/keystone/changeset/accept_ownership.go

```diff
@@ -0,0 +1,86 @@
+package changeset
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	ccipowner "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/changeset"
+)
+
+// AcceptAllOwnershipsProposal creates a MCMS proposal to call accept ownership on all the Keystone contracts in the address book.
+func AcceptAllOwnershipsProposal(e deployment.Environment, chainSelector uint64, minDelay time.Duration) (deployment.ChangesetOutput, error) {
```

Comment:
change this signature to match the generic changeset func and add a type assertion [eg](https://github.com/smartcontractkit/chainlink/blob/86d5f9993ef208ef0a310a781b198563f4ae7d55/deployment/keystone/changeset/append_node_capbilities.go#L11)